### PR TITLE
Cancel placeholder for search input and update snapshot

### PR DESCRIPTION
**Pull request by w8896699**
Instead of using placeholder, we are now using searching query from last time as value after click search(Intimate what other search engine is doing).

### CODE REVIEW

File: app-web/src/components/Search/index.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ export const Search = ({ onSearch, searchOnEnter, inputConfig, query, ...rest })
         type="text"
         query={query}
         aria-label={ARIA_LABEL_SEARCH_INPUT}
-        value={terms}
+        value={terms.useState}
```
Comment (patricksimonian) [inline app-web/src/components/Search/index.js]: where `.useState` coming from? `terms` should be fine on its own, I think what you are looking for is to initially set terms with a query if its available. In that case, you could use a `useEffect(() => {...}, [])` to accomplish this.

File: app-web/src/components/Search/index.js [hunk 2]
```diff
@@ -49,9 +49,10 @@ export const Search = ({ onSearch, searchOnEnter, inputConfig, query, ...rest })
 
   const search = () => {
     onSearch(terms);
-    setTerms('');
   };
 
+  const handleFocus = event => event.target.select();
```
Comment (patricksimonian) [inline app-web/src/components/Search/index.js]: what does this do?

File: app-web/src/utils/helpers.js [hunk 3]
```diff
@@ -61,11 +61,13 @@ export const getTextAndLink = (resourceType, resourcesByType) => {
   return textAndPath;
 };
 
-/** return a combine text result or results depens on how much search results we have
+/** return a combine text result or results depends on how much search results we have.
```
Comment (patricksimonian) [inline app-web/src/utils/helpers.js]: i'd think this top line of documentation is redundant

File: app-web/src/components/Search/index.js (hunk 2 above)
Comment (w8896699) [reply app-web/src/components/Search/index.js]: This will allow single click and select all text in input box

Comment (patricksimonian) [reply app-web/src/components/Search/index.js]: hmm I'm not sure if that will be an effect we will want 100% of the time when searching. Just my opinion. @sheaphillips ?

I feel like if you have 'searched' than a single click highlight would be an awesome UX improvement. If it happens always, it prevents users from 'fixing typos'

Comment (patricksimonian) [reply app-web/src/components/Search/index.js]: Also as a tip, i'd add a tiny comment above that event handler that explains exactly what the function does  :)

**Pull request merged**